src/standalone-mocks/notifications-mock.tsx

```
@@ -1,31 +1,18 @@
 import React from 'react';
 
-export const addNotification = (
-  notification: Record<string, unknown>,
-): void => {
+export const addNotification = (notification: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)